Hunting Queries/AWSS3/AWSBucketAPILogs-SuspiciousDataAccessToS3BucketsfromUnknownIP.yaml

```diff
@@ -20,9 +20,9 @@ query: |
   let endtime = todatetime('{{EndTimeISO}}');
   let lookback = starttime - 14d;
   AwsBucketAPILogs_CL 
-  | where EventTime TimeGenerated between(starttime..endtime)
+  | where EventTime between(starttime..endtime)
```

Comment:
Is this query working? According to the schema of AwsBucketAPILogs_CL, EventName field will not be there. I think we meant AWSS3BucketAPILogParsed in first part of join as well.